### ISSUE TITLE
Make data from 'Număr de cazuri pe zile' graph more accessible

### ISSUE THREAD

**Issue by aniri** [code4romania/date-la-zi#324]
The data shown in the 'Număr de cazuri pe zile'  should be made available to screen readers. 

Suggestion: add a separate hidden table that the screen reader can pick up. Add all graph data in the table.

**Comment by mihainsto** [comment]
I can take this. :blush:

**Comment by CristiHabliuc** [comment]
Closed via #325
